src/OrchardCore.Modules/OrchardCore.Localization/ModularPoFileLocationProvider.cs

```diff
@@ -39,18 +39,32 @@ public class ModularPoFileLocationProvider : ILocalizationFileLocationProvider
         public IEnumerable<IFileInfo> GetLocations(string cultureName)
         {
             var poFileName = cultureName + PoFileExtension;
+            var extensions = _extensionsManager.GetExtensions();
 
             // Load .po files in each extension folder first, based on the extensions order
-            foreach (var extension in _extensionsManager.GetExtensions())
+            foreach (var extension in extensions)
```

Comment:
That's ok, here the file provider is initialized with `ContentRootFileProvider` which in our case is a composite of the original one and our embedded file provider.

            env.ContentRootFileProvider = new CompositeFileProvider(
                new ModuleEmbeddedFileProvider(appContext),
                env.ContentRootFileProvider);

Reply:
Yes I think all the App_Data folders are explicitly removed from source because of some rule set in one of the targets files. This code path would make sense only for people working with source code.

Reply:
Also while debugging I saw that there's already one pass that looks for "fr" and a second one that looks for "fr-CA" but need to double check that.